KachnaOnline.Business/Exceptions/BoardGames/UserNotFoundException.cs

```diff
@@ -0,0 +1,14 @@
+// UserNotFoundException.cs
+// Author: František Nečas
+
+using System;
+
+namespace KachnaOnline.Business.Exceptions.BoardGames
```

Comment:
This exception is generally useful for all subsystems in the project. I would highly suggest generalizing the exception to use in events and states planning as well. There is already the same exception in [states branch](https://github.com/ondryaso/kachna-online/blob/states/KachnaOnline.Business/Exceptions/UserNotFoundException.cs) (currently used for states planning) generalized for everyone to use. That is a better design in my opinion.
```suggestion
namespace KachnaOnline.Business.Exceptions
```